client/js/table-view.js

```diff
@@ -37,24 +38,39 @@ class TableView {
 	renderTable() {
 		this.renderTableHeader();
 		this.renderTableBody();
+		this.renderTableFooter();
 	}
 
 	renderTableHeader() {
 		removeChildren(this.headerRowEl)
-
 		getLetterRange('A', this.model.numCols)
 			.map(colLabel => createTH(colLabel))
-			// .forEach(things => console.log('things ', things));
 			.forEach(th => this.headerRowEl.appendChild(th));
 	}
 
+
+
 	isCurrentCell(col, row) {
 		return this.currentCellLocation.col === col
 		 && 
 		 	   this.currentCellLocation.row === row; 
 
 	}
 
+	renderTableFooter() {
+		const footerFragment = document.createDocumentFragment();
```

Comment:
Rendering the table footer is a lot like rendering the table header, so you could use the renderTableHeader function as inspiration for this one. How would this look without creating a document fragment?